frontend/src/lib/api.ts

```diff
@@ -1430,6 +1436,7 @@ const api = {
             headers: {
                 ...(isFormData ? {} : { 'Content-Type': 'application/json' }),
                 'X-CSRFToken': getCookie(CSRF_COOKIE_NAME) || '',
+                // 'X-POSTHOG-SESSION-ID': posthog.get_session_id(),
```

Comment:
Should these headers be commented out?

Reply:
Well done, you both spotted my deliberate mistake 🏅 

![baby-circle](https://github.com/PostHog/posthog/assets/984817/235e171c-7d41-46d2-8125-8ebbcd6386d5)